### PR TITLE
Add mobile navbar

### PR DESCRIPTION
**Pull request by Tanya-atatakai**
image
image

### CODE REVIEW

**Comment by oaksprout** [comment]
@Tanya-atatakai I noticed another issue – when you click on Get Involved, it follows the link correctly, but the menu doesn't close.

File: components/Layout/Footer.jsx [hunk 1]
```diff
@@ -141,25 +141,27 @@ const LinksBlock = ({ title, links, className }) => (
...
-          key={item.title}
-          className="hover:text-black"
-          {...(item.isExternal
-            ? {
-              target: '_blank',
-              rel: 'noopener noreferrer',
-            }
-            : {})}
-        >
+        <span key={item.title} className="contents">
```
Comment (Tanya-atatakai) [inline components/Layout/Footer.jsx:144]: this is just fixing not aligned footer, noticed that and couldn't pass by....
image

File: components/Layout/MenuMobile.jsx [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+                  </NavigationMenu.Trigger>
+                </NavigationMenu.Item>
+
+                <NavigationMenu.Item>
+                  <NavigationMenu.Trigger asChild>
+                    <Link href="/explore" className={linkClassName}>
+                      Explore
+                    </Link>
+                  </NavigationMenu.Trigger>
+                </NavigationMenu.Item>
```
[the remark this answers is not in the document]
Comment (Tanya-atatakai) [reply components/Layout/MenuMobile.jsx:48]: you're right, will update that

[the remark this answers is not in the document]
Comment (Tanya-atatakai) [reply components/Layout/MenuMobile.jsx]: fixed

**Comment by Tanya-atatakai** [comment]
> when you click on Get Involved, it follows the link correctly, but the menu doesn't close.
@oaksprout fixed

**Review by oaksprout — APPROVED** [review]

**Review by truemiller — APPROVED** [review]

**Pull request merged**